webpack.icons.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
         "index": "./src/clarity-icons/index.ts",
         "interfaces/icon-alias": "./src/clarity-icons/interfaces/icon-alias.ts",
         "interfaces/icon-template": "./src/clarity-icons/interfaces/icon-template.ts",
+        "clarity-icons-api": "./src/clarity-icons/clarity-icons-api.ts",
```

Comment:
@goodmorninggoaway thanks for submitting this PR. Could you also add `clarity-icons-element`?
```
"clarity-icons-element": "./src/clarity-icons/clarity-icons-element.ts"
```